### PR TITLE
Add Short Link

### PR DESCRIPTION
**Pull request by FayasNoushad**
`https://shortlink.api.fayas.me/?query=<sample-url>`

`https://shortlink.api.fayas.me/?query=https://github.com`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1175,6 +1175,7 @@ API | Description | Auth | HTTPS | CORS |
 | [T.LY](https://t.ly/docs) | URL shortener API | No | Yes | No |
 | [TinyUID](https://tinyuid.com/docs) | Shorten long URLs | No | Yes | Yes |
 | [Zero Width Shortener](https://docs.zws.im) | Shortens URLs using spaces that have zero width, making them invisible to humans | No | Yes | Unknown |
+| [All Shortener](https://shortlink.api.fayas.me) | Short URLs support so many domainsdomains | No | Yes | Unknown |
```
Comment (matheusfelipeog) [inline README.md:1178]: Use your project name in the title: Short Link.
Use a URL that goes to your project's documentation.
Fix API description.

```suggestion
| [Short Link](https://github.com/FayasNoushad/Short-Link-API) | Short URLs support so many domains | No | Yes | Unknown |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @FayasNoushad 😄 

Can you resolve this review, please?

Also, don't forget to fix the alphabetical order.

**Comment by matheusfelipeog** [comment]
Now fix the alphabetical order so that I accept your API, please. ;)

**Comment by FayasNoushad** [comment]
> Now fix the alphabetical order so that I accept your API, please. ;)

Done

**Review by matias904 — APPROVED** [review]

**Comment by FayasNoushad** [comment]
> Now it's Ok. 
> 
> A big THANKS for your update @FayasNoushad ;)

Always welcome Mr. @matheusfelipeog

**Pull request merged**